practice_05.11.19_classes/Task1_Figures/Figure.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace practice_05._11._19_classes.Task1_Figures
+{
+    public class Figure
+    {
+        public double Ordinate { get; set; }
```

Comment:
I supposed to make these member protected for access from inheritors only.

Reply:
fixed